pkg/controller/installation/products/amqstreams/reconciler.go

```diff
@@ -45,17 +47,17 @@ type Reconciler struct {
 	mpm           marketplace.MarketplaceInterface
 }
 
-func (r *Reconciler) Reconcile(inst *v1alpha1.Installation, serverClient pkgclient.Client) (v1alpha1.StatusPhase, error) {
+func (r *Reconciler) Reconcile(inst *v1alpha1.Installation, serverClient pkgclient.Client, ctx context.Context) (v1alpha1.StatusPhase, error) {
```

Comment:
generally considered good practice to have context as the first argument

Reply:
sure, I'll move the arg list